Rust/rust/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[workspace]
+
+members = [
+    "Samples/allapis",
+    "Samples/ADC/adc_high_level_app",
+    "Samples/DHCP",
+    "Samples/CertStore/cert_store_high_level_app",
+    "Samples/CustomNTP/custom_ntp_high_level_app",
+    "Samples/GPIO/gpio_high_level_app",
+    "Samples/HelloWorld/hello_world_high_level_app"
+]
+
+default-members = [
+    "Samples/allapis",
```

Comment:
Nit: `Samples` is not the normal naming convention in Rust. `examples` would be the normal name. Of course, that's if you want to run those examples through cargo so perhaps a different name is the right way to go. In any case, the directory name should be lowercase.

Reply:
Samples and examples are a bit different concepts.  Will change the case of the directory before pushing.... the name matches the Samples directory in the Sphere C SDK.